rts/Lua/LuaOpenGL.cpp

```diff
@@ -4147,6 +4154,52 @@ int LuaOpenGL::Clear(lua_State* L)
 	return 0;
 }
 
+namespace Impl {
+	template<class Type, auto glClearBufferFuncPtrPtr>
+	static inline void ClearBuffer(lua_State* L, GLenum bufferType, GLint drawBuffer) {
+		Type values[4];
+		values[0] = luaL_optnumber(L, 3, 0);
```

Comment:
Add `spring::SafeCast<Type>` to each assignment, just in case

Reply:
Ok.